picard/file.py

```diff
@@ -709,12 +709,12 @@ def update(self, signal=True):
                         self.state = File.CHANGED
                     break
             else:
-                if (self.metadata.images
+                self.similarity = 1.0
```

Comment:
I wonder if we should have a constant (`MAXSIMILARITY`?) instead of `1.0`

Reply:
Not really sure about that specific case. We have multiple places where we deal with scoring between 0.0 and 1.0 (basically percentages). They are similar because of the range, would need to be checked if they are actually always the same and would reuse the same constant.

But actually because this is such a clear concept I think a constant doesn't help readability here. Quite the opposite, reading ` 1.0`  makes it more visible what's happening than a constant. It's the same as if we had e.g. percentage values between 0 and 100. I'd find reading a `100`  in code much clearer than a constant `ONE_HUNDRED_PERCENT`.

I'd see it differently if we had some other strange max, such as `0.97` or so. Then I'd definitely argue for a constant.